packages/client/src/check-deployment-status.ts

```diff
@@ -95,7 +95,7 @@ export async function* checkDeploymentStatus(
       ) {
         yield {
           type: 'warning',
```

Comment:
Let's change the type here to `alias-warning` so that way we don't have two shapes for the same type